extensions/amp-skimlinks/0.1/skim-options.js

```diff
@@ -14,10 +14,19 @@
  * limitations under the License.
  */
 
-import {GLOBAL_DOMAIN_BLACKLIST} from './constants';
+import {getChildJsonConfig} from '../../../src/json';
 import {getNormalizedHostnameFromUrl} from './utils';
 import {userAssert} from '../../../src/log';
 
+import {
```

Comment:
rather than importing this list of constants, can we move `defaultConfig` in constants and just import that. As well as move the `errors` constant below.